File: assets/samples/dns.lookup/password/python/dns_lookup_pass.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

domain = 'example.com'
password = 'your whois api password'
username = 'your whois api username'

url = 'http://www.whoisxmlapi.com/whoisserver/DNSService?type=_all'\
    + '&domainName=' +domain + '&username=' +username + '&password=' +password

print(urlopen(url).read().decode('utf8'))

File: assets/samples/dns.lookup/password/php/dns_lookup_pass.php
<?php

$domain = 'example.com';
$password = 'your whois api password';
$username = 'your whois api username';

$url ="http://www.whoisxmlapi.com/whoisserver/DNSService?domainName={$domain}"
     ."&username={$username}&password={$password}&type=A,SOA,TXT";

print(file_get_contents($url));